test/marine/CMakeLists.txt

```diff
@@ -176,6 +184,10 @@ if (GENERATE_BUFR2IODA_TESTS)
 	ADD_INSITU_TEST("profile_glider" "subpfl")
 	ADD_INSITU_TEST("profile_tesac" "tesac")
 	ADD_INSITU_TEST("profile_tropical" "dbuoy")
+	ADD_INSITU_TEST("profile_rama" "mbuoyb")
+	ADD_INSITU_TEST("profile_pirata" "mbuoyb")
+	ADD_INSITU_TEST("profile_taotriton" "mbuoyb")
+	ADD_INSITU_TEST("profile_ndbc" "mbuoyb")
```

Comment:
Nitpicky, but most (if not all) NCBD buoys are reporting surface variables. There is no profile, generally. So a more appropriate name would be "surface_ndbc"

Reply:
That's a good point @JohnSteffen-NOAA . Can you fix this here and in obsForge @givelberg .